packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1110,6 +1137,9 @@ export class ContainerRuntime extends EventEmitter
 
     public process(messageArg: ISequencedDocumentMessage, local: boolean) {
         this.verifyNotClosed();
+        if (this.racingToSummarize) {
```

Comment:
Why does it need special treatment? I.e. why we do not pause ops same way we do when generating regular summaries?

Reply:
We still want to watch for summaryAck ops so we can know when it's safe to refresh.